benchmarks/htap/lib/monitoring.py

```diff
@@ -29,33 +29,44 @@ def _print(self):
         self.lines = []
 
     def display_summary(self, elapsed):
-        elapsed_seconds = max(1, elapsed.total_seconds())
-        tps = self.stats.oltp_total('ok')    / elapsed_seconds
-        eps = self.stats.oltp_total('error') / elapsed_seconds
-        tpmc = trunc((self.stats.oltp_total('new_order') / elapsed_seconds) * 60)
-        num_queries, num_errors, num_timeouts = self.stats.olap_totals()
-        throughput = num_queries * 3600 / elapsed_seconds
         print()
         summary = 'Summary'
         print(f'{summary}\n' + len(summary) * '-')
         print(f'Scale factor: {self.num_warehouses // WAREHOUSES_SF_RATIO }')
         print(f'Workers: {self.num_oltp_workers} OLTP, {self.num_olap_workers} OLAP')
+
+        elapsed_seconds = max(1, elapsed.total_seconds())
         print(f'Total time: {elapsed_seconds:.2f} seconds')
-        print(f'OLTP AVG Transactions per second (TPS): {tps:.2f}')
-        print(f'OLTP AVG Errors per second: {eps:.2f}')
-        print(f'OLTP New-Order transactions per minute (tpmC): {tpmc:.0f}')
-        print(f'OLAP Throughput (queries per hour): {throughput:.1f}')
-        print(f'OLAP Errors {num_errors:}, Timeouts: {num_timeouts:}')
+        # TODO output time after burn-in step
+
+        print('')
+        print(f'OLTP Average transactions per second (TPS)')
```

Comment:
how about instead reporting both the TPS and latency?
or just the same table that we output in the monitor?

Reply:
I can add the latency. I also thought about the whole table, but:
1- the last state of the table is exactly what you would see here
2- it stops being a summary (which I'd like it to be concise)

Reply:
Added, and starting to use tabulate there too